src/components/TextEditor/components/TextEditor.js

```diff
@@ -34,6 +36,10 @@ class TextEditor extends React.Component {
       codeMirrorInstance: null,
       currentLine: 0,
       sketch: null,
+      showModal: false,
```

Comment:
Quick comment: this should probably be `showForkModal` just because we might/will add more modals later!